<launch>
  <arg name="range" default="2"/>
  <arg name="gain_percent" default="100"/>
  <arg name="gamma" default="127"/>
  <arg name="ping_rate" default="0"/>
  <arg name="freq_mode" default="2"/>
  <arg name="ip_address" default="auto"/>
  <arg name="send_gain" default="true"/>
  <arg name="data_size" default="8bit"/>
  <arg name="frame_id" default="oculus"/>
  <node pkg="oculus_sonar_driver" exec="oculus_driver_node" name="oculus_driver" output="screen">
    <param name="range" value="$(arg range)"/>
    <param name="gain" value="$(arg gain_percent)"/>
    <param name="gamma" value="$(arg gamma)"/>
    <param name="ping_rate" value="$(arg ping_rate)"/>
    <param name="freq_mode" value="$(arg freq_mode)"/>
    <param name="ip_address" value="$(arg ip_address)"/>
    <param name="send_gain" value="$(arg send_gain)"/>
    <param name="data_size" value="$(arg data_size)"/>
    <param name="frame_id" value="$(arg frame_id)"/>
    <param from="$(find oculus_sonar_driver)/config/oculus_params.yaml" />
  </node>
</launch>
